docs/control-structures.rst

```diff
@@ -324,12 +324,14 @@ In the following example, we show how ``throw`` can be used to easily revert an
 
 Currently, there are six situations, where exceptions happen automatically in Solidity:
 
-1. If you access an array beyond its length (i.e. ``x[i]`` where ``i >= x.length``).
-2. If a function called via a message call does not finish properly (i.e. it runs out of gas or throws an exception itself).
-3. If a non-existent function on a library is called or Ether is sent to a library.
-4. If you divide or modulo by zero (e.g. ``5 / 0`` or ``23 % 0``).
-5. If you perform an external function call targeting a contract that contains no code.
-6. If a contract-creation call using the ``new`` keyword fails.
+1. If you access an array on or beyond its length (i.e. ``x[i]`` where ``i >= x.length``) or below zero.
```

Comment:
Would `with a negative index` easier to get than `or below zero`? Perhaps even make it a second line.